### ISSUE TITLE
Using with Angular 12 not working

### ISSUE THREAD

**Issue by MurhafSousli** [amcharts/amcharts5#28]
When used with Angular 12, it throws the following errors

```
Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapChart.d.ts:192:35 - error TS2503: Cannot find namespace 'GeoJSON'.

192     protected _geometryColection: GeoJSON.GeometryCollection;
                                      ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapLine.d.ts:8:16 - error TS2503: Cannot find namespace 'GeoJSON'.

8     geometry?: GeoJSON.LineString | GeoJSON.MultiLineString;
                 ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapLine.d.ts:8:37 - error TS2503: Cannot find namespace 'GeoJSON'.

8     geometry?: GeoJSON.LineString | GeoJSON.MultiLineString;
                                      ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapLineSeries.d.ts:19:16 - error TS2503: Cannot find namespace 'GeoJSON'.

19     geometry?: GeoJSON.LineString | GeoJSON.MultiLineString;
                  ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapLineSeries.d.ts:19:37 - error TS2503: Cannot find namespace 'GeoJSON'.

19     geometry?: GeoJSON.LineString | GeoJSON.MultiLineString;
                                       ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapLineSeries.d.ts:61:29 - error TS2503: Cannot find namespace 'GeoJSON'.

61     protected _types: Array<GeoJSON.GeoJsonGeometryTypes>;
                               ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPointSeries.d.ts:12:16 - error TS2503: Cannot find namespace 'GeoJSON'.

12     geometry?: GeoJSON.Point | GeoJSON.MultiPoint;
                  ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPointSeries.d.ts:12:32 - error TS2503: Cannot find namespace 'GeoJSON'.

12     geometry?: GeoJSON.Point | GeoJSON.MultiPoint;
                                  ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPointSeries.d.ts:92:29 - error TS2503: Cannot find namespace 'GeoJSON'.

92     protected _types: Array<GeoJSON.GeoJsonGeometryTypes>;
                               ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPointSeries.d.ts:102:61 - error TS2503: Cannot find namespace 'GeoJSON'.

102     protected _positionBulletReal(bullet: Bullet, geometry: GeoJSON.Geometry, coordinates: [number, number], angle?: number): void;
                                                                ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPolygon.d.ts:9:16 - error TS2503: Cannot find namespace 'GeoJSON'.

9     geometry?: GeoJSON.MultiPolygon | GeoJSON.Polygon;
                 ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPolygon.d.ts:9:39 - error TS2503: Cannot find namespace 'GeoJSON'.

9     geometry?: GeoJSON.MultiPolygon | GeoJSON.Polygon;
                                        ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPolygonSeries.d.ts:15:16 - error TS2503: Cannot find namespace 'GeoJSON'.

15     geometry?: GeoJSON.Polygon | GeoJSON.MultiPolygon;
                  ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPolygonSeries.d.ts:15:34 - error TS2503: Cannot find namespace 'GeoJSON'.

15     geometry?: GeoJSON.Polygon | GeoJSON.MultiPolygon;
                                    ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapPolygonSeries.d.ts:43:29 - error TS2503: Cannot find namespace 'GeoJSON'.

43     protected _types: Array<GeoJSON.GeoJsonGeometryTypes>;
                               ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapSeries.d.ts:5:16 - error TS2503: Cannot find namespace 'GeoJSON'.

5     geometry?: GeoJSON.Geometry;
                 ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapSeries.d.ts:6:20 - error TS2503: Cannot find namespace 'GeoJSON'.

6     geometryType?: GeoJSON.GeoJsonGeometryTypes;
                     ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapSeries.d.ts:13:15 - error TS2503: Cannot find namespace 'GeoJSON'.

13     geoJSON?: GeoJSON.GeoJSON;
                 ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapSeries.d.ts:58:29 - error TS2503: Cannot find namespace 'GeoJSON'.

58     protected _types: Array<GeoJSON.GeoJsonGeometryTypes>;
                               ~~~~~~~


Error: node_modules/@amcharts/amcharts5/.internal/charts/map/MapSeries.d.ts:59:24 - error TS2503: Cannot find namespace 'GeoJSON'.

59     _geometries: Array<GeoJSON.Geometry>;
                          ~~~~~~~

```

Any idea how to fix these errors?

**Comment by MurhafSousli** [comment]
I fixed it by installing the types for the following d3 packages

```
    "@types/d3-chord": "^3.0.1",
    "@types/d3-force": "^3.0.3",
    "@types/d3-geo": "^3.0.2",
    "@types/d3-hierarchy": "^3.0.2",
    "@types/d3-sankey": "^0.11.2",
    "@types/d3-shape": "^3.0.2",
```

**Comment by martynasma** [comment]
Fixed in [5.0.0-beta.30](https://github.com/amcharts/amcharts5/releases/tag/5.0.0-beta.30).

## [5.0.0-beta.30] - 2021-10-05

### Added
- `shadow*` settings are now available on `Label` as well.
- `arrangeTooltips` (default: `true`) setting added to `XYChart`. If set to `false` will disable tooltip overlap check algo.

### Fixed
- Fixed type errors when importing maps (especially in Angular).
- Fixed `templateField` to override `template`.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download-v5/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
